README.rst

```diff
@@ -50,9 +50,54 @@ We encourage macOS developers to use `homebrew <https://brew.sh/>`_ to install a
 
     /usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
     brew update
-    brew install python pypy3 gettext awscli postgresql
+    brew install python pypy3 gettext awscli postgresql socat
     brew link gettext --force
-    brew cask install docker
+    # We need to install a specific version of docker since newer ones have a bug around the builtin proxy
+    https://raw.githubusercontent.com/caskroom/homebrew-cask/61f1d33be340e27b91f2a5c88da0496fc24904d3/Casks/docker.rb
+
+After installing Docker, open it, and navigate to Preferences -> Daemon. There add ``172.30.0.0/16`` to the list of insecure registries, then click ``Apply and Restart``.
+
```

Comment:
We probably also want to tell users to go into Docker's preferences and *uncheck* General -> "Automatically check for updates" since we are targeting the specific old version.